### PR TITLE
HDDS-11464. Removed unused constants from OzoneConsts.

### PR DESCRIPTION
**Pull request by weimingdiit**
## What changes were proposed in this pull request?
Removed some unused constants from OzoneConsts.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-11464

## How was this patch tested?
unit tests

### CODE REVIEW

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java [hunk 1]
```diff
@@ -101,20 +96,15 @@ public final class OzoneConsts {
...
   public static final String CONTAINER_TEMPORARY_CHUNK_PREFIX = "tmp";
   public static final String CONTAINER_CHUNK_NAME_DELIMITER = ".";
-  public static final String CONTAINER_ROOT_PREFIX = "repository";
 
   public static final String FILE_HASH = "SHA-256";
   public static final String MD5_HASH = "MD5";
   public static final String CHUNK_OVERWRITE = "OverWriteRequested";
 
-  public static final int CHUNK_SIZE = 1 * 1024 * 1024; // 1 MB
+  public static final int CHUNK_SIZE = 1024 * 1024;
```
Comment (ayushtkn) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java:107]: that 1 might be deliberately added for better understandability, maybe we can let this line stay as is

**Review by sarvekshayr — COMMENTED** [review]
Thanks @weimingdiit for working on this patch.
We also have `OZONE_REQUEST` and `OM_S3_VOLUME_PREFIX` unused in `OzoneConsts`, should we go ahead and remove them as well?

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java (hunk 1 above)
Comment (weimingdiit) [reply hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java]: @ayushtkn  Thanks for your comment, I have updated the code, please review it again.

**Comment by weimingdiit** [comment]
> Thanks @weimingdiit for working on this patch. We also have `OZONE_REQUEST` and `OM_S3_VOLUME_PREFIX` unused in `OzoneConsts`, should we go ahead and remove them as well?

@sarvekshayr  Thanks for your comment, I have updated the code, please review it again.

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java [hunk 2]
```diff
@@ -101,20 +94,15 @@ public final class OzoneConsts {
...
   public static final String CONTAINER_TEMPORARY_CHUNK_PREFIX = "tmp";
   public static final String CONTAINER_CHUNK_NAME_DELIMITER = ".";
-  public static final String CONTAINER_ROOT_PREFIX = "repository";
 
   public static final String FILE_HASH = "SHA-256";
   public static final String MD5_HASH = "MD5";
   public static final String CHUNK_OVERWRITE = "OverWriteRequested";
 
-  public static final int CHUNK_SIZE = 1 * 1024 * 1024; // 1 MB
+  public static final int CHUNK_SIZE = 1 * 1024 * 1024;
```
Comment (sarvekshayr) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java:105]: Please include this as well.
```suggestion
  public static final int CHUNK_SIZE = 1 * 1024 * 1024; // 1 MB
```

**Review by sarvekshayr — APPROVED** [review]
Thank you @weimingdiit.
LGTM. Waiting for green CI.

**Review by ayushtkn — APPROVED** [review]
LGTM

**Comment by ayushtkn** [comment]
Merged, Thanx @weimingdiit for the contribution & @sarvekshayr for the review!!!

**Comment by weimingdiit** [comment]
@ayushtkn Thanks for your review and merge. & @sarvekshayr  thanks for the review!

**Pull request merged**